install/installer/pkg/components/ws-manager/role.go

```diff
@@ -16,6 +16,32 @@ func role(ctx *common.RenderContext) ([]runtime.Object, error) {
 	labels := common.DefaultLabels(Component)
 
 	return []runtime.Object{
+		&rbacv1.ClusterRole{
+			TypeMeta: common.TypeMetaClusterRole,
+			ObjectMeta: metav1.ObjectMeta{
+				Name:      Component,
+				Namespace: ctx.Namespace,
+				Labels:    labels,
+			},
+			Rules: []rbacv1.PolicyRule{
+				{
+					APIGroups: []string{"snapshot.storage.k8s.io"},
+					Resources: []string{
+						"volumesnapshotcontents",
```

Comment:
Question: do we need to grant `volumesnapshotcontents` as well?

Reply:
you mean volumesnapshot? that one is granted as regular role, since it is namespaced object.

Reply:
Sorry, typo, I'm asking for the VolumeSnapshotClass.

Reply:
Ah. So far no need. If in the future we will need it, we can always add it (at least that my way of thinking on it).